pkg/systemd/generate/systemdgen.go

```diff
@@ -164,6 +164,17 @@ func CreateContainerSystemdUnit(info *ContainerInfo, opts Options) (string, erro
 			"--cidfile", "%t/%n-cid",
 			"--cgroups=no-conmon",
 		}
+
+		// Check detach param
```

Comment:
Please add a more verbose comment why we want to force `--detach`.

Reply:
thanks for the guide. I've forced pushed the PR